rundeckapp/grails-app/views/menu/home.gsp

```diff
@@ -62,9 +62,6 @@
 
 	<asset:javascript src="menu/home.js"/>
 
-	<!-- VUE CSS MODULES -->
-	<asset:stylesheet href="static/css/components/version-notification.css"/>
-	<!-- /VUE CSS MODULES -->
```

Comment:
This file isn't being created. In fact the components in the folder version-notification do not seem to be used anywhere.